.github/workflows/publish.yml

```
@@ -68,3 +68,10 @@ jobs:
     secrets: inherit
     with:
       image_tag: ${{ inputs.image_tag }}
+
+  publish-worker-v4:
+    needs: [typecheck, units]
+    uses: ./.github/workflows/publish-worker-v4.yml
```

actionlint (1.7.4)

could not read reusable workflow file for "./.github/workflows/publish-worker-v4.yml": open /home/jailuser/git/.github/workflows/publish-worker-v4.yml: no such file or directory

(workflow-call)